spec/jekyll-feed_spec.rb

```diff
@@ -224,6 +224,14 @@
         expect(feed_meta).not_to include('title=')
       end
     end
+
+    context "with an invalid site URL" do
+      let(:overrides) { { "url" => "example.org" } }
```

Comment:
Because there is no `http://` or `https://`

If this appeared in an `href`, it would be interpreted as `./example.org` which breaks things.